.github/workflows/ci.yml

```diff
@@ -86,18 +86,22 @@ jobs:
         run: npm install ajv
       - name: linter
         run: node ./utils/reply_schema_linter.js
+        # TODO: delete everything after this line
```

Comment:
Suggest keeping, and run it only in case json files are changed, or if explicitly required (we can add a parameter for that)

Reply:
See `on:push:path` pattern:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
and: https://docs.github.com/en/actions/learn-github-actions/contexts#inputs-context for inputs

Reply:
mmm idk also feels like overkill (it takes even longer than valgrind, which we also run daily)
on the other hand, changes to the jsons are less likely to happen so the extra time won't affect most PRs

@oranagra ?

Reply:
the inputs one is irrelevant since AFAIK it's not possible to simply get a button in the PR to trigger it the CI with an additional input.

we do use the workflow_dispatch in daily.yml, so we can run this on demand on any branch (i've built a mechanism there that let's you input a repo+branch pair).

regarding the push:path filter that's nice, maybe we can use it to condition the linter execution.
but i'm afraid to let the full test run automatically per PR CI for two reasons:
1. it won't get triggered just by schema changes, any change in the json files will trigger it (even a description or arity change).
2. the test takes way too long (why is that btw? maybe we can speed it up?)